src/main/java/com/epam/page/object/generator/builder/StringUtils.java

```diff
@@ -10,20 +10,25 @@ public static String splitCamelCase(String camel) {
         String result = (trim.charAt(0) + "").toLowerCase();
         int spaces = 0;
         for (int i = 1; i < trim.length(); i++) {
-            String letter = trim.charAt(i)+"";
+            String letter = trim.charAt(i) + "";
             if (letter.equals(" ")) {
-                if (++spaces == 3) return result; }
-            else
-                result += trim.charAt(i-1) == ' '
-                        ? letter.toUpperCase()
-                        : letter.toLowerCase();
+                if (++spaces == 3) {
```

Comment:
Are you sure if you haven't broken a logic of this method? Also, is it possible to avoid ++ operator and magic numbers?

Reply:
It's only code style refactoring